lib/sidekiq/pool/cli.rb

```diff
@@ -111,6 +112,14 @@ def parse_options(argv)
             @pool_config = arg
           end
 
+          o.on '-m', '--working-direcory PATH', "path to working direcory" do |arg|
```

Comment:
Typo in `direcory`

Not sure about `-m`. Other good options like `-d` are taken. Maybe just have long version?